frontend/Container.js

```diff
@@ -19,6 +19,15 @@ var TabbedPane = require('./TabbedPane');
 
 import type MenuItem from './ContextMenu';
 
+type Props = {};
+
+type State = {
+  isVertical: boolean,
+};
+
+var IS_VERTICAL_BREAKPOINT = 500;
+var resizeTimeout = null;
```

Comment:
The linter didn't like the `?:` syntax, so I just set it's initial value to null